pkg/component/component.go

```diff
@@ -423,10 +421,18 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 // Build component from BuildConfig.
 // If 'wait' is true than it waits for build to successfully complete.
 // If 'wait' is false than this function won't return error even if build failed.
-func Build(client *occlient.Client, componentName string, applicationName string, wait bool, stdout io.Writer) error {
+// 'show' will determine whether or not the log will be shown to the user
+func Build(client *occlient.Client, componentName string, applicationName string, wait bool, show bool, stdout io.Writer) error {
 
+	// No loading spinner if we're showing the logging output
 	// Loading spinner
-	s := log.Spinnerf("Triggering build from git")
+
+	var s *log.Status
+	if show {
```

Comment:
why not use `show` in `log.Spinnerf` directly?
`s := log.Spinnerf(!show, "Building component")`